pkg/utils/gopdf.go

```diff
@@ -80,7 +80,7 @@ const (
 	// defaultTtfFontPath is the default path to ttf font
 	// stsong.ttf downloaded from https://www.wfonts.com/font/stsong
 	// Reason to use this font: it supports Chinese(https://github.com/signintech/gopdf/issues/68)
-	defaultTtfFontPath = "./ttf/stsong.ttf"
+	defaultTtfFontPath = "../../resource/ttf/stsong.ttf"
```

Comment:
Maybe `resource/ttf/stsong.ttf`.
If it is `../../resource/ttf/stsong.ttf`, then the program should run in the `ttf` directory to find the relevant resources by default.

Reply:
But currently there is no effect on the output of the pdf.